test/examples/integer/integer_primes.jl

```diff
@@ -9,16 +9,16 @@ function test_integer_primes()
         ᾱ = 1
```

Comment:
an example with the \bar{} not rendering right.

Reply:
It is dependent on which fonts the browser (or vscode) loads. I agree that accentuation is this tabu-spiced deadly unicode sin and should be changed.